packages/vendure-plugin-payment-extensions/src/payment-extensions-plugin.ts

```diff
@@ -0,0 +1,23 @@
+import { PluginCommonModule, VendurePlugin } from '@vendure/core';
+import { settleWithoutPaymentHandler } from './settle-without-payment-handler';
+import { settleWithoutPaymentChecker } from './settle-without-payment-checker';
+
+@VendurePlugin({
+  imports: [PluginCommonModule],
+  configuration: (config) => {
+    config.paymentOptions.paymentMethodHandlers.push(
+      settleWithoutPaymentHandler
+    );
+    if (config.paymentOptions.paymentMethodEligibilityCheckers?.length) {
```

Comment:
Is this check really needed? I thought these arrays where always defined. If not, this is fine

Reply:
I believe so, `config.paymentOptions.paymentMethodEligibilityCheckers` is `PaymentMethodEligibilityChecker<ConfigArgs>[] | undefined`